data/fields/cuisine.json

```
@@ -16,16 +16,23 @@
             "kebab": "Kebab",
             "indian": "Indian",
             "chicken": "Chicken",
+            "fish": "Fish",
             "asian": "Asian",
             "french": "French",
+            "spanish": "Spanish",
+            "portugese": "Portugese",
+            "russian": "Russian",
+            "polish": "Polish",
             "thai": "Thai",
+            "korean": "Korean",
             "american": "American",
             "sushi": "Sushi",
```

If we're adding fish, we should definitely add the broader seafood (and probably fish_and_chips as well):
```suggestion
            "seafood": "Seafood",
            "sushi": "Sushi",
```